src/api/timeld.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+require_once(__DIR__ . '/../../vendor/autoload.php');
+require_once(__DIR__ . '/callGetEndpoint.php');
+require_once(__DIR__ . '/callEndpoint.php');
+
+
+function importTimld($data = array()) {
+    $url = $_SERVER["TIMELD_HOST"] . '/import';
+
+
+    $headers = array(
+        "Accept: Content-Type: application/x-ndjson",
```

Comment:
Should this be `Accept: application/x-ndjson`?

Reply:
Or if this is export *to* Timeld, maybe it should be `Content-Type: application/x-ndjson`?